lib/perl/Genome/SoftwareResult.pm

```diff
@@ -1190,4 +1190,22 @@ sub expunge {
     return 1;
 }
 
+sub resize_disk_allocation {
```

Comment:
Ideally we don't need both this and `Genome::SoftwareResult::Stageable::_reallocate_disk_allocation`.